react-carousel-component/my-app/src/ArrowButton.js

```diff
@@ -0,0 +1,18 @@
+import './ArrowButton.css';
+import { BsArrowBarLeft, BsArrowBarRight } from 'react-icons/bs';
+
+export default function ArrowButton({ isForward, onCustomClick }) {
```

Comment:
You could just use `onClick` instead of `onCustomClick`, as long as you don't get confused! In the exercises I used `onCustomClick` because students kept confusing that with button's `onClick`.